source/io/module_httpClient.js

```diff
@@ -625,6 +629,19 @@
                 Module.eggShell.setOccurrenceAsync(occurrencePointer);
             });
         };
+
+        Module.httpClient.jsHttpClientSetCredentials = function (handle, isWithCredentials, errorStatusPointer, errorCodePointer, errorSourcePointer) {
+            var errorStatus = Module.eggShell.dataReadBoolean(errorStatusPointer);
+            if (errorStatus) {
+                return;
+            }
+
+            var httpClient = findhttpClientOrWriteError(handle, 'LabVIEWHTTPClient:Credentials', errorStatusPointer, errorCodePointer, errorSourcePointer);
+            if (httpClient === undefined) {
+                return;
+            }
+            httpClient.setCredentials(isWithCredentials !== 0);
```

Comment:
There are several named values of 0 for this file, please pick a descriptive one:
https://github.com/ni/VireoSDK/blob/incoming/source/io/module_httpClient.js#L29